### DISCUSSION TITLE
Example of CompoundFilter()

### DISCUSSION THREAD

**Discussion by victorterancas** [jheddings/notional#55] [Q&A] [answered]
Hello 👋 ,

I am trying to use the `CompoundFilter()` function. This is the furthest i have gone:
`        query = (
            self._notional.databases.query(databaseID)
            .filter(CompoundFilter.or_())`
I found this [user guide](https://jheddings.github.io/notional/query/). But it doesn't explain about this topic.

Thank you.

Btw, do you have a telegram group or channel?

**Comment by jorgeromanespino** [comment, marked as answer]
Hi Victor!

This works for me. 
query = (
    notion.databases.query(dbid)
        .filter(filter=CompoundFilter(or_=[
            PropertyFilter(property='Categoría', select=SelectCondition(equals="Instalación")),
            PropertyFilter(property='Categoría', select=SelectCondition(equals="Zona"))
        ]))
)

**Reply by victorterancas** [reply]
Thanks, it works perfectly. 😄.

**Reply by jheddings** [reply]
Great response!  Seeing it written makes me think we can do better with some helpers for composing filters...  🤔

**Reply by victorterancas** [reply]
Not working anymore.

**Reply by jheddings** [reply]
@victorterancas can you provide more information and fill out a bug report?  I'm able to use the `CompoundFilter` without any issues.  For reference, here is a query that is executing correctly:

```python
query = (
    notion.databases.query(dbid)
    .filter(
        filter=CompoundFilter(
            or_=[
                PropertyFilter(
                    property="Title", rich_text=TextCondition(contains="project")
                ),
                PropertyFilter(
                    property="Cost", number=NumberCondition(greater_than=1000000)
                ),
            ]
        )
    )
    .limit(1)
)
```

**Comment by jheddings** [comment]
Looking into this...  There have been some considerable changes in the API over the past couple of months.  It might take a while to get caught up.  😒

**Reply by victorterancas** [reply]
I totally understand. It was my mistake 🙈, i was trying to filter using "string" while i should use "title". I realise when it happens the same problem with [ramnes/notion-sdk-py](https://github.com/ramnes/notion-sdk-py).

**Reply by jheddings** [reply]
Got it!  Thanks for confirming.
